src/utils.rs

```diff
@@ -109,15 +109,25 @@ pub fn walk_ptrs_ty<'t>(ty: ty::Ty<'t>) -> ty::Ty<'t> {
 ///         }
 ///     }
 #[macro_export]
-macro_rules! if_let_chain {
```

Comment:
Why rename the macro? It's a chain of `if let`s, not `if`s, the two aren't interchangeable.